# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/box-shadow/611574-1.html
# Commit: 2684fdcd6fbf
# Full Hash: 2684fdcd6fbff5abe54a3e1c018e7188abb4076c
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 09:34:48
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r e29eecc5147c -r 2684fdcd6fbf layout/reftests/box-shadow/611574-1.html
--- a/layout/reftests/box-shadow/611574-1.html	Mon Nov 25 15:28:50 2019 +0000
+++ b/layout/reftests/box-shadow/611574-1.html	Mon Nov 25 21:45:22 2019 +0000
@@ -1,9 +1,10 @@
 <!DOCTYPE HTML>
 <html><head><style>
-button,span,fieldset { background-color: #0F0; }
+button,span,fieldset { background-color: #0F0; break-inside:avoid; }
+body { font-size:12px; }
 </style></head>
-<body style="margin:30px">
-<div style="column-count:3;">
+<body>
+<div style="column-count:3;height:600px;column-fill:auto">
 <p><button style="box-shadow: inset -20px 10px 0 #0F0;">No Blur Radius</button>
 <p><button style="box-shadow: inset  20px 10px 0 #0F0;">No Blur Radius</button>
 <p><button style="box-shadow: inset -20px 10px 0 4px #0F0;">No Blur Radius</button>